MainRepository/Program.cs

```diff
@@ -1,7 +1,56 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
+﻿var pq = new PriorityQueue<int> ();
+pq.EnQueue (25);
+pq.EnQueue (8);
+Console.WriteLine (pq.DeQueue ());
+pq.EnQueue (27);
+pq.EnQueue (3);
+Console.WriteLine (pq.DeQueue ());
+pq.EnQueue (1);
+pq.EnQueue (2);
+pq.EnQueue (7);
+foreach (var item in pq.mData)
+   Console.Write (item + " ");
+Console.WriteLine ();
+
+/// <summary>Implementation of PriroityQueue using Binary Min Heap</summary>
+public class PriorityQueue<T> where T : IComparable<T> {
+
+   public List<T> mData = new();
+
+   /// <summary>Adds element in the priority queue</summary>
+   public void EnQueue (T item) {
+      mData.Add (item);
+      int ci = mData.Count - 1;

```

Comment:
Bad naming of the variable
```suggestion
      int child = mData.Count - 1;

```